src/borg/testsuite/archiver/__init__.py

```diff
@@ -195,6 +195,11 @@ def create_regular_file(self, name, size=0, contents=None):
         filename = os.path.join(self.input_path, name)
         if not os.path.exists(os.path.dirname(filename)):
             os.makedirs(os.path.dirname(filename))
+        self.write_to_file(filename, size=size, append_path=False, contents=contents)
+
+    def write_to_file(self, filename, size=0, append_path=True, contents=None):
+        if append_path:
+            filename = os.path.join(self.input_path, filename)
```

Comment:
that's a good idea to save the callers from having to prepend the input path.

but guess that should be:
- refactored globally in a separate PR
- done by default for `create_regular_file`, not adding an extra method

Reply:
hmm, considering that tests often check like `assert "input/somefile" in output` it is maybe not a good idea to remove this "input" part at the caller location.